specs/core/1_shard-data-chains.md

```diff
@@ -299,78 +331,201 @@ def compute_crosslink_data_root(blocks: Sequence[ShardBlock]) -> Hash:
     return hash_tree_root(BytesN[MAX_SIZE](pad(header + footer, MAX_SIZE)))
 ```
 
+### `get_default_shard_state`
+
+```python
+def get_default_shard_state(beacon_state: BeaconState, shard: Shard) -> ShardState:
+    earlier_committee = get_period_committee(
+        beacon_state,
+        Epoch(PHASE_1_FORK_SLOT - SHARD_SLOTS_PER_BEACON_SLOT * SLOTS_PER_EPOCH * EPOCHS_PER_SHARD_PERIOD * 2),
```

Comment:
Missing shard slot -> epoch conversion.

```suggestion
        Epoch(
            (PHASE_1_FORK_SLOT - SHARD_SLOTS_PER_BEACON_SLOT * SLOTS_PER_EPOCH * EPOCHS_PER_SHARD_PERIOD * 2) //
            SHARD_SLOTS_PER_BEACON_SLOT * SLOTS_PER_EPOCH
        ),
```

Reply:
What do you think about adding a helper function for `ShardSlot` -> `Epoch` conversion?

Reply:
We already have one! `compute_epoch_of_shard_slot`